sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -314,16 +384,23 @@ public String generateSas(TableSasSignatureValues tableSasSignatureValues) {
     }
 
     /**
-     * Inserts an entity into the table if it does not exist, or merges the entity with the existing entity otherwise.
+     * Inserts an {@link TableEntity entity} into the table if it does not exist, or merges the
+     * {@link TableEntity entity} with the existing {@link TableEntity entity} otherwise.
+     *
+     * If no {@link TableEntity entity} exists within the table having the same partition key and row key as the
+     * provided {@link TableEntity entity}, it will be inserted. Otherwise, the provided {@link TableEntity entity}'s
+     * properties will be merged into the existing {@link TableEntity entity}.
```

Comment:
Isn't this say the same thing as the paragraph above it? If we want to keep one, I'd choose this one.